web/test/pages/profile/profile-starting.test.tsx

```diff
@@ -1329,6 +1329,98 @@ describe("profile - starting business", () => {
     });
   });
 
+  describe("profile opportunities alert for Poppy", () => {
+    const ProfileConfig = Config.profileDefaults.fields;
+
+    it("only displays alert on info tab", () => {
+      const business = generateBusinessForProfile({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          businessPersona: "STARTING",
+          dateOfFormation: undefined,
+        }),
+      });
+      renderPage({ business });
+      expect(screen.getByTestId("opp-alert")).toBeInTheDocument();
+      chooseTab("numbers");
+      expect(screen.queryByTestId("opp-alert")).not.toBeInTheDocument();
+    });
+
+    it("lists each unanswered funding/certification question", () => {
```

Comment:
[boulder] We have the same duplicative tests repeated three times over for each of these personas. It seems like everything would be a lot easier to read if we instead had a unit test for `ProfileOpportunitiesAlert` that described the specific behavior of the alert. It would have the following benefits:
- No need to copy-paste and duplicate the same tests across personas because one test describes the behavior
- When things DO differ by persona, like for Foreign, makes it much more obvious and easier to understand
- Simplifies the profile tests. The only tests that would need to remain in profile would be about which tab it shows on, and what happens to it when fields get interacted with, because all that is within a larger context than just the alert and its unit test

Reply:
taken out